src/chapters/razor-views/using-templates.rst

```diff
@@ -0,0 +1,90 @@
+Using a Template
+================
+
+This page and the ones that follow introduce how to start to pass data between the MVC elements.
+We haven't addressed models yet (those come next) so this just means sharing information between 
+the controller and view. 
+
+.. index:: ! ViewBag
+
+Passing Data to a Template
+--------------------------
+
+The controller class contains methods that send data to different templates.
+These methods have a structure similar to:
+
+.. sourcecode:: c#
+   :linenos:
+
+   public IActionResult ActionMethod()
+   {
+      // method code here
+      
+      ViewBag.property = Data;
+      return View();
+   }
+
+**ViewBag** is an object that passes data into a template. ``Data`` can
+be a variable of any type, a number, a collection of some sort, or an object. 
+A ``ViewBag`` property is created and given a value as simply as is done on line 5 above.
```

Comment:
I think this explanation could be expanded. Right now, it reads like ViewBag is a set system object with set properties and if I found the right page in the docs I could see the list of available properties. This [article](https://www.tutorialsteacher.com/mvc/viewbag-in-asp.net-mvc) on ViewBag had a few points that really helped me figure out what ViewBag is, such as it is dynamic and it is used to pass data from the controller into the view. Also, it is a dynamic dictionary so knowing that helped me realize that I could name the properties whatever I wanted to because the properties act as the keys.